src/nix/nix_store.rs

```diff
@@ -128,9 +128,11 @@ impl NixStoreCmd {
             Ok(out)
         } else {
             let exit_code = out.status.code().unwrap_or(1);
+            let stderr_output = String::from_utf8(out.stderr)?;
```

Comment:
Should this not be `from_utf8_lossy`?

Reply:
```
?;
```

Pretty much this causes the function fail on decoding errors with stderr, which is something we don't care about. Lossy conversation is fine.

Reply:
As an aside: must be *capture* stderr? Why not let the parent process inherit and stream it directly to the user (still in stderr)?